content/cli/commands/produce.md

```diff
@@ -221,13 +226,71 @@ partitions using [`fluvio partition list`].
 %copy first-line%
 ```bash
 $ fluvio partition list
- TOPIC          PARTITION  LEADER  REPLICAS  RESOLUTION  HW  LEO  LSR  FOLLOWER OFFSETS
- multi-no-keys  0          0       []        Online      2   2    0    []
- multi-no-keys  1          0       []        Online      2   2    0    []
- multi-no-keys  2          0       []        Online      2   2    0    []
- multi-no-keys  3          0       []        Online      2   2    0    []
- multi-no-keys  4          0       []        Online      2   2    0    []
+ TOPIC          PARTITION  LEADER  REPLICAS  RESOLUTION  SIZE   HW  LEO  LRS  FOLLOWER OFFSETS 
+ multi-no-keys  0          0       []        Online      120 B  2   2    0    [] 
+ multi-no-keys  1          0       []        Online      121 B  2   2    0    [] 
+ multi-no-keys  2          0       []        Online      124 B  2   2    0    [] 
+ multi-no-keys  3          0       []        Online      126 B  2   2    0    [] 
+ multi-no-keys  4          0       []        Online      127 B  2   2    0    [] 
 ```
 
 Notice how the high watermark (HW) and log-end-offset (LEO) tell us that there are
 exactly 2 records in each partition. Our ten records have been evenly distributed!
+
+## Example 4: Producing using a compression algorithm (GZIP)
+
+Fluvio support different types of compression algorithms to send records. 
+Compression, in general, improves throughput in exchange of some CPU cost to compress/uncompress the data.
+
+Let's try to use `gzip` algorithm in the CLI.
+
+First, we'll use [`fluvio topic create`] to create a topic called `gzip` and other topic called `none`:
+
+[`fluvio topic create`]: {{< ref "/cli/commands/topic#fluvio-topic-create" >}}
+
+%copy first-line%
+```bash
+$ fluvio topic create gzip
+```
+
+Next, let's create a text file with the records we want to send:
+
+```bash
+# Put the following records into a text file using your favorite editor
+$ cat records.txt
```

Comment:
Close ``` here and add %copy first-line% at the top.

Then open a new bash

%copy%
```bash
...
```
That way users can copy/paste the command they need to run, as well as the content.

Reply:
I think that it is useless the ability to copy `cat records.txt`, and I think that having that in a separate block will look strange

![image](https://user-images.githubusercontent.com/22335041/158206418-3a1f7e1f-d5f8-48a0-b6ae-27bd997d1c45.png)

Reply:
We used this format before. Add test for the file, then give the ability to copy/paste.
<img width="985" alt="image" src="https://user-images.githubusercontent.com/26554085/158212629-f37b89a3-30e8-4c32-9569-3bcf29a3416d.png">